src/alarms/notifications/NewNotification.js

```diff
@@ -1,120 +1,41 @@
-
 import React, { Component } from "react";
-
-import { addNotification } from "../../actions";
 import { connect } from "react-redux";
 import { withRouter } from "react-router-dom";
-import { FormattedMessage, injectIntl } from "react-intl";
-import { Map, Marker, TileLayer, WMSTileLayer } from "react-leaflet";
-import ErrorOverlay from "../../data_management/rasters/ErrorOverlay.js";
-
-import ManagementForm from "../../forms/ManagementForm";
-import TextInput from "../../forms/TextInput";
-
-class NewNotification extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      isFetching: false,
-      openOverlay: false,
-      modalErrorMessage: "",
-      createdNotification: null,
-    };
-  }
-
-  handleResponse(response) {
-    this.setState({
-      modalErrorMessage: response,
-      isFetching: false,
-      handlingDone: true
-    });
-  }
+import MDSpinner from "react-md-spinner";
 
-  onSubmit = (validatedData, currentNotification, alarmType) => {
-    this.setState({ isFetching: true, openOverlay: true });
+import { NotificationForm } from "./NotificationForm";
 
-    const url = `/api/v4/${alarmType}`;
-  }
-  
+class NewNotificationModel extends Component {
   render() {
-    const position = [52.1858, 5.2677];
-    const { currentNotification } = this.props;
-
-    return (
-      <div>
-        {this.state.openOverlay ? (
-          <ErrorOverlay
-            isFetching={this.state.isFetching}
-            history={this.props.history}
-            errorMessage={this.state.modalErrorMessage}
-            handleClose={() =>
-              this.setState({ handlingDone: false, openOverlay: false })}
-            currentNotification={this.props.currentNotification || this.state.createdNotification}
-          />
-        ) : null}
-        <ManagementForm
-
-        >
-          <TextInput
-          name="notificationName"
-          title="Name of this alarm"
-          />
-          <TextInput
-          name="typeSelection"
-          title="Source type selection"
-          />
-          <TextInput
-          name="rasterSelection"
-          title="Raster selection"
-          />
-          <TextInput
-          name="timeseriesSelection"
-          title="Timeseries selection"
-          />
-          <TextInput
-          name="relativeStart"
-          title="Relative start"
-          />
-          <TextInput
-          name="relativeEnd"
-          title="Relative end"
-          />
-          <TextInput
-          name="thresholds"
-          title="Alarm thresholds"
-          />
-          <TextInput
-          name="snoozing"
-          title="Snoozing"
-          />
-          <TextInput
-          name="recipients"
-          title="Recipients"
-          />
-        </ManagementForm>
+    if (
+      this.props.organisations.isFetching === false
```

Comment:
Why does this depend on organisations being fetched?

Reply:
It does not depend on organisations being fetched. So I removed this part.